sender_test.go

```diff
@@ -186,17 +187,18 @@ func TestInt64Serialization(t *testing.T) {
 
 func TestLong256Column(t *testing.T) {
 	ctx := context.Background()
-
+	
 	testCases := []struct {
 		name     string
-		val      string
+		val     *big.Int
 		expected string
 	}{
-		{"random bits ", "0x0123a4i", "0x0123a4"},
-		{"8bit max", "0xffffffffi", "0xffffffff"},
-		{"16bit max", "0xffffffffffffffffi", "0xffffffffffffffff"},
-		{"32bit max", "0xffffffffffffffffffffffffffffffffi", "0xffffffffffffffffffffffffffffffff"},
-		{"64bit long", "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffi", "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"},
+		{"0", big.NewInt(0).SetInt64(0), "0x00"},
```

Comment:
Don't we need to test `0x00i`, not `0x00` here and below?

Reply:
The expected string is `0x00` because QDB's valid output also shows the same. The test fails when the suffix is added `0x00i`
Same goes for other examples as well.

Reply:
Sorry, I haven't noticed appended 'i' suffix on L219.